src/cql/r4/FHIRCommon.cql

```diff
@@ -0,0 +1,94 @@
+library FHIRCommon version '3.0.0'
+
+/*
+FHIR-specific common functionality for CQL
+*/
+
+using FHIR version '4.0.1'
```

Comment:
@mldorr @ivan-c noting here that we are switching to FHIR version **4.0.1**, I think it is worth trying loading the app first **without** the new MME functions to make sure things don't break.  Just concerned that the frontend library, "cql-exec-fhir", that is used by AHRQ app to translate CQL here https://github.com/uwcirg/AHRQ-CDS-Connect-PAIN-MANAGEMENT-SUMMARY/blob/78e5f7a9b77d31df3ffabd1d9aa5a9ea573cee35/src/utils/executeELM.js#L3)
does not include FHIR model info for version 4.0.1:  https://github.com/cqframework/cql-exec-fhir/tree/master/src/modelInfos  

Steps I took to test this out:
- commenting out any code related to new MME functions in src/cql/r4/Factors_to_Consider_in_Managing_Chronic_Pain_FHIRv400.cql  including line **1354** where **ReportMME** is called
- regenerate associated JSON file, i.e, src/cql/r4/Factors_to_Consider_in_Managing_Chronic_Pain_FHIRv400.json (following steps listed by Maggie here:  https://docs.google.com/document/d/1EOVhwEzEwXlMWY9SDQjNxTLR6OOrKQUdo41TE9KCyS4/edit

Application loaded but with missing information:
![Screen Shot 2020-07-24 at 9 35 29 AM](https://user-images.githubusercontent.com/12942714/88414100-2dc41a80-cd91-11ea-803f-a2bfb613cb58.png)
And I saw some runtime errors (see screenshot):
<img width="987" alt="Screen Shot 2020-07-24 at 9 05 23 AM" src="https://user-images.githubusercontent.com/12942714/88411697-3adf0a80-cd8d-11ea-8f7a-87eec26704f2.png">
But let me know what you see when you try this out, just want to comment here so you are aware of this possible issue.